css/style.css

```diff
@@ -1 +1,42 @@
+body {
+	font-family: "Open Sans", sans-serif;
+	padding: 40px;
+	background-color: #fafafa;
+}
 
+a {
+	text-decoration: none;
+	color: #129BB0;
+}
+
+img {
+	border-radius: 50%;
+	max-width: 300px;
+}
+
+ul.nav {
+	margin: 0px;
```

Comment:
You actually don't need to follow 0s with a px. This is up to the developer's preference, but being the minimalist I am these could be deleted.